File: swagger/models/get_session_types_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetSessionTypesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination properties used.

		session_types
		Contains information about sessions.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'SessionTypes': 'session_types',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'session_types': 'SessionTypes',
		}

	pagination_response = None
	session_types = None

File: swagger/Models/GetSessionTypesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetSessionTypesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination properties used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about sessions.
	 * 
	 * @var SessionType[] 
	 */
	public $sessionTypes;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'SessionTypes' => ['sessionTypes', 'array', SessionType::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'sessionTypes' => ['SessionTypes', 'array', SessionType::class],
			];
	}
}
